src/control/server/ctl_storage_rpc.go

```diff
@@ -159,32 +184,32 @@ func (c *StorageControlService) StorageScan(ctx context.Context, req *ctlpb.Stor
 	msg := "Storage Scan "
 	resp := new(ctlpb.StorageScanResp)
 
-	controllers, err := c.NvmeScan()
+	bsr, err := c.bdev.Scan(bdev.ScanRequest{})
 	if err != nil {
 		resp.Nvme = &ctlpb.ScanNvmeResp{
 			State: newState(c.log, ctlpb.ResponseStatus_CTL_ERR_NVME, err.Error(), "", msg+"NVMe"),
 		}
 	} else {
 		resp.Nvme = &ctlpb.ScanNvmeResp{
 			State:  newState(c.log, ctlpb.ResponseStatus_CTL_SUCCESS, "", "", msg+"NVMe"),
-			Ctrlrs: controllers,
+			Ctrlrs: nvmeControllersToPB(c.log, bsr.Controllers),
```

Comment:
I want to use the FromNative() method from proto.NvmeControllers here, but the Healthstats misalignment issues prevent that for now. In the interests of keeping this PR focused, I'll refrain from fixing that now. Will work on that ASAP after this one though.